src/components/sidebar/sidebar.tsx

```diff
@@ -21,6 +24,12 @@ export default function Sidebar({ className, ...rest }: SidebarProps) {
     dispatch(setTiers([]));
   };
 
+  const handleReset = () => {
+    dispatch(setTiles([]));
```

Comment:
Reset should not put default values. It should remove existing tiles from rhs and place in lhs. It must include tiles that are added by users. Frameworks and sample tiers are just for reference. 

Please, add a method in slice to unassign tiers from the tiles.

Reply:
can you please provide some hints on this